Add hello-world template

joshka opened the pull request:
Simplify the Ratatui readme to two commands.

```shell
cargo install cargo-generate
cargo generate ratatui/templates
```

orhun commented:
I rebased this on `main` for later doing on a rebase-merge to not lose the atomic commits. However I see only 2 commits on this branch now, can you check? 🤔

joshka commented:
> I rebased this on `main` for later doing on a rebase-merge to not lose the atomic commits. However I see only 2 commits on this branch now, can you check? 🤔

I had not pulled the latest master, so that was my bad. Fixed, rebased to just one commit and pushed.
